### PR TITLE
introduce GitHub release notes prescription job

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Closes: https://github.com/thoth-station/prescriptions/issues/18

### CODE REVIEW

File: prescriptions-gh-release-notes-job/base/openshift-templates/adviser.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        #           prescriptions_gh_release_notes.yaml"
+        #         endpoint: "{{inputs.parameters.THOTH_S3_ENDPOINT_URL}}"
+        #         bucket: "{{inputs.parameters.THOTH_CEPH_BUCKET_NAME}}"
+        #         insecure: true
+        #         accessKeySecret:
+        #           name: argo-artifact-repository-secrets
+        #           key: accessKey
+        #         secretKeySecret:
+        #           name: argo-artifact-repository-secrets
+        #           key: secretKey
```
Comment (fridex) [inline prescriptions-gh-release-notes-job/base/openshift-templates/adviser.yaml:90]: We will need to use `resource` here to propagate values from configmap, see https://github.com/argoproj/argo-workflows/issues/3212#issuecomment-642824124

File: prescriptions-gh-release-notes-job/base/openshift-templates/prescriptions-gh-release-notes.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+    - name: prescriptions-gh-release-notes
+      inputs:
+        parameters:
+          - name: bucket-name
+          - name: bucket-prefix
+          - name: deployment-name
+          - name: s3-endpoint-url
+      container:
+        image: quay.io/thoth-station/prescriptions-gh-release-notes-job:v0.0.2
```
[the remark this answers is not in the document]
Comment (harshad16) [reply prescriptions-gh-release-notes-job/base/openshift-templates/prescriptions-gh-release-notes.yaml:90]: ack , thanks @fridex :+1:

**Review by harshad16 — COMMENTED** [review]
/lgtm 
once the wip is removed , we can get this merged.

**Comment by goern** [comment]
@fridex is this still WIP?

**Comment by fridex** [comment]
> @fridex is this still WIP?

Yes, the job is ready to be deployed.

**Pull request merged**